src/asvz_bot.py

```diff
@@ -408,46 +426,64 @@ def __get_enrollment_and_start_time(driver):
 
     @staticmethod
     def __get_enrollment_time(driver):
-        enrollment_interval_raw = driver.find_element(
-            By.XPATH, "//dl[contains(., 'Einschreibezeitraum')]/dd"
-        )
-        # enrollment_interval_raw is like 'So, 09.05.2021 06:35 - Mo, 10.05.2021 07:05'
-        enrollment_start_raw = (
-            enrollment_interval_raw.get_attribute("innerHTML")
-            .split("-")[0]
-            .split(",")[1]
-            .strip()
-        )
         try:
-            enrollment_start = datetime.strptime(enrollment_start_raw, "%d.%m.%Y %H:%M")
-        except ValueError as e:
-            logging.error(e)
-            raise AsvzBotException(
-                "Failed to parse enrollment start time: '{}'".format(
-                    enrollment_start_raw
-                )
+            enrollment_interval_raw = driver.find_element(
+                By.XPATH, "//dl[contains(., 'Einschreibezeitraum')]/dd"
+            )
+            # enrollment_interval_raw is like 'So, 09.05.2021 06:35 - Mo, 10.05.2021 07:05'
+            enrollment_start_raw = (
+                enrollment_interval_raw.get_attribute("innerHTML")
+                .split("-")[0]
+                .split(",")[1]
+                .strip()
             )
+            try:
+                enrollment_start = datetime.strptime(enrollment_start_raw, "%d.%m.%Y %H:%M")
+            except ValueError as e:
+                logging.error(e)
+                raise AsvzBotException(
+                    "Failed to parse enrollment start time: '{}'".format(
+                        enrollment_start_raw
+                    )
+                )
+        except NoSuchElementException as e:
+            logging.info("No enrollment time found. Assuming that enrollment is open.")
+            enrollment_start = datetime.today()
         return enrollment_start
```

Comment:
As you nicely added a log statement to `__get_lesson_time`, I thought we could use one here as well 😇 

```suggestion
        logging.info("Enrollment starts at {}".format(enrollment_start.strftime("%H:%M:%S")))
        return enrollment_start
```